7.1/Dockerfile

```diff
@@ -55,13 +57,15 @@ RUN mkdir /tmp/sessions && \
     chown -R 1001:0 /var/log/nginx && \
     chown -R 1001:0 /var/lib/nginx && \
     chown -R 1001:0 /opt/app-root /tmp/sessions && \
+    chown -R 1001:0 /etc/php-fpm.d && \
```

Comment:
this is needed so that a custom s2i assemble can change the php-fpm.d folder if it wants to.